docs/intro/tutorial.rst

```diff
@@ -625,12 +625,12 @@ attribute automatically. So the code can be shortened further::
     for a in response.css('li.next a'):
         yield response.follow(a, callback=self.parse)
 
-.. note::
+To create multiple requests from an iterable, you can use
```

Comment:
This explanation is a bit confusing, because `li.next a` should match a single link in most cases. 

A more natural example would be something like `response.css('.author ~ a')` - follow all links to authors - though the selector could be too complex for a tutorial (or maybe not?).

Reply:
Yeah, I looked at the look and did not noticed the actual CSS expression. May I remove the look as I revert back to `follow`?

Reply:
Thanks @Gallaecio! 

There is a use case for a loop though (and for follow_all on a single link), as it works on a last page, where there is no "next" link; new code (with `[0]`) fails with exception. 

I'm on fence on whether this pattern is good or not (use follow_all for cases where 0 or 1 result is expected). I can see myself using this pattern in spider code, but it can make the code less readable.

Reply:
To be honest, I don't remember if we agreed on something specific here, but the following is a somewhat verbose `IndexError`-safe alternative using `response.follow`:
```python
a = response.css('li.next a')
if a:
    yield response.follow(a[0], callback=self.parse)
```

Reply:
Replaced `li.next` occurrences with `ul.pager`, which is still consistent with the source page. Hopefully it's clearer now, let me know what you think.

![Screenshot at 2020-01-22 10-41-14](https://user-images.githubusercontent.com/1731933/72898912-c0f99600-3d03-11ea-9ae3-029f38387360.png)